server/views/formPages/openConditions/furtherCharges.html

```diff
@@ -18,7 +18,7 @@
 {% endblock %}
 
 {% set furtherChargesHtml %}
-  {% if catType == 'INITIAL' %}
+  {% if catType == 'INITIAL'  and data.ratings.furtherCharges.furtherCharges == 'Yes' %}
```

Comment:
Is it nullsafe? Can any elements in any scenarios in`data.ratings.furtherCharges` be null/undefined? 
Same question applies to line 66 below as well.

Reply:
it was tested

Reply:
Do you mean 
{% if catType == 'INITIAL'  and data.ratings.furtherCharges.furtherCharges **===** 'Yes' %}
rather than 
{% if catType == 'INITIAL'  and data.ratings.furtherCharges.furtherCharges **==** 'Yes' %}
?

Reply:
nope we will not do a strict check, we are going to just use loose check, going along with catType == 'INITIAL'